compass/landice/tests/thwaites/run_model.py

```diff
@@ -0,0 +1,96 @@
+from compass.model import make_graph_file, run_model
+from compass.step import Step
+
+
+class RunModel(Step):
+    """
+    A step for performing forward MALI runs as part of Thwaites test cases.
+
+    Attributes
+    ----------
+    suffixes : list of str, optional
+        a list of suffixes for namelist and streams files produced
+        for this step.  Most steps most runs will just have a
+        ``namelist.landice`` and a ``streams.landice`` (the default) but
+        the ``restart_run`` step of the ``restart_test`` runs the model
+        twice, the second time with ``namelist.landice.rst`` and
+        ``streams.landice.rst``
+    """
+    def __init__(self, test_case, name='run_model', subdir=None,
+                 cores=1, min_cores=None, threads=1, suffixes=None):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        name : str, optional
+            the name of the test case
+
+        subdir : str, optional
+            the subdirectory for the step.  The default is ``name``
+
+        cores : int, optional
+            the number of cores the step would ideally use.  If fewer cores
+            are available on the system, the step will run on all available
+            cores as long as this is not below ``min_cores``
+
+        min_cores : int, optional
+            the number of cores the step requires.  If the system has fewer
+            than this number of cores, the step will fail
+
+        threads : int, optional
+            the number of threads the step will use
+
+        suffixes : list of str, optional
+            a list of suffixes for namelist and streams files produced
+            for this step.  Most steps most runs will just have a
+            ``namelist.landice`` and a ``streams.landice`` (the default) but
+            the ``restart_run`` step of the ``restart_test`` runs the model
+            twice, the second time with ``namelist.landice.rst`` and
+            ``streams.landice.rst``
```

Comment:
```suggestion
            for this step.  Most run steps will just have a
            ``namelist.landice`` and a ``streams.landice`` (the default) but
            the ``restart_run`` step of the ``restart_test`` runs the model
            twice, the second time with ``namelist.landice.rst`` and
            ``streams.landice.rst``
```
I'm guessing this typo is in all `RunModel` steps in `landice`. Sorry about that!